client/src/Components/PostCreator/PostCreator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import { Form } from 'react-bootstrap';
+import { useHistory } from 'react-router';
+
+const PostCreator = (props) => {
+	let history = useHistory();
+
+	function handleClick(e) {
+		e.preventDefault();
+		let formElement = document.querySelector('form');
+		var formData = new FormData(formElement);
+		let title = formData.get('titleInput');
+		let body = formData.get('bodyInput');
```

Comment:
Just a random thought: is it necessary to use the word `Input` for the element's name?